Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net471</TargetFrameworks>
+    <TargetFrameworks>net6.0;net471;net472</TargetFrameworks>

```

Comment:
is it cleanup or are we adding new supported version?

Reply:
should we add `net471/2` also to our CI builds e.g. in https://github.com/snowflakedb/snowflake-connector-net/pull/754

Reply:
This version is already supported, it was just missing in the target framework statement thus it was not tested on the job.

We are already building and testing it within `test-windows` job:
```
      matrix:
        dotnet: ['net6.0', 'net472', 'net471']
```